From 9e1f9908662ebefad5932bda5bb96a648ae140ed Mon Sep 17 00:00:00 2001
From: Hannes Hornwall <hannes@hornwall.me>
Date: Sun, 8 Apr 2018 21:57:44 +0200
Subject: [PATCH] Add hybrid colors

---
 config.def.h | 38 +++++++++++++++++++-------------------
 1 file changed, 19 insertions(+), 19 deletions(-)

diff --git a/config.def.h b/config.def.h
index 2b55389..5e8717c 100644
--- a/config.def.h
+++ b/config.def.h
@@ -83,29 +83,29 @@ static char termname[] = "st-256color";
 static unsigned int tabspaces = 8;
 
 /* bg opacity */
-static const int alpha = 0xdd;
+static const int alpha = 0xF2;
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	"#1e1f22",
+	"#d1434c",
+	"#99ac4c",
+	"#f3bd5a",
+	"#688fb3",
+	"#ab78ad",
+	"#68b5ab",
+	"#5c656e",
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#161719",
+	"#e37c44",
+	"#3f7b4d",
+	"#947023",
+	"#4f4775",
+	"#4b3148",
+	"#68b5ab",
+	"#b7bcba",
 
 	[255] = 0,
 
@@ -120,8 +120,8 @@ static const char *colorname[] = {
  * Default colors (colorname index)
  * foreground, background, cursor, reverse cursor
  */
-static unsigned int defaultfg = 7;
-static unsigned int defaultbg = 257;
+static unsigned int defaultfg = 15;
+static unsigned int defaultbg = 8;
 static unsigned int defaultcs = 256;
 static unsigned int defaultrcs = 257;
 
-- 
2.16.3

